docs/JavaScript/README.md

```diff
@@ -106,29 +106,45 @@ const has_crypto = false;
 <a id="import-rules-require"></a>
```

Comment:
closed this PR due to the conflicts: https://github.com/binary-com/deriv-app/pull/9999